labshare-client/src/views/UeberUns.vue

```diff
@@ -1,12 +1,14 @@
-  }
-}
+    {
+    "en": {
+    "welcome": "Welcome to LabHive!",
+    "aboutUs": "<h1>Welcome to LabHive!</h1><p>We support diagnosic centers, scientists and technical staff with the improvment of their SARS-CoV-2 diagnostic. LabHive promotes communication and connects</p><ul>    <li>qualified volunteers from science with the closest diagnostic laboratory looking for support</li>    <li>Diagnostic centers with research laboratories for equipment and reagent exchange</li>    <li>Diagnostic centers with scientists providing special Know-How</li></ul><hr /><p>In early 2020 the new SARS-CoV-2 virus spread accross the globe. The viral infection is widespread and can cause COVID-19, an acute respiratory disease that can cause severe and long-lasting symptoms especially in older patients and patients with preexisting diseases. In the worst case the patients require ventilation in an intensive care unit. Therefore the danger of overwhelming the healthcare system, even in developed countries, with the large number of patients arises, leading to an increase in expected mortality. Patients are often already infectious before showing any symptoms, adding to the observed high infection rate. In order to track the spread within the population, put infected patients under quarantine and end quarantine measures after recovery, RT-qPCR tests are used. These tests are able to detect the virus in nose and throat swabs in the case of infection. In order to increase test capacities and master the infection wave in the best possible way ressources like qualified personel, machines and reagents are missing. Our platform conveys between qualified volunteers, closed research laboratories with unused machines and unused reagents and diagnostic centers, so that the diagnostic can uphold and increase their capacities. Spread the test, not the virus!</p>"
```

Comment:
"improvment of their SARS-CoV-2 diagnostic" -> "improvement of their diagnostic capacity"

"Know-How" -> "know-how"

"Therefore the danger of overwhelming the healthcare system, even in developed countries, with the large number of patients arises, leading to an increase in expected mortality" -> "Therefore the danger arises of overwhelming the healthcare system with large numbers of patients, even in developed countries, leading to an increase in expected mortality"

"test capacities" -> "test capacity"

"master the infection wave" -> "flatten the curve"

"ressources" -> "resources"

"are missing" -> "are needed"

"conveys between" -> "connects"

"so that the diagnostic can uphold and increase their capacities" -> "so that diagnostic capabilities can be increased"